notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -80,9 +80,9 @@
        "      <th>sex</th>\n",
```

Comment:
<p>a</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>&lt;del_n7bd&gt;&lt;div class=\"toc\"&gt;&lt;ul class=\"toc-item\"&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Export-Training-Samples\" data-toc-modified-id=\"Export-Training-Samples-1\"&gt;Export Training Samples&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Data-Preparation\" data-toc-modified-id=\"Data-Preparation-2\"&gt;Data Preparation&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Model-Training\" data-toc-modified-id=\"Model-Training-3\"&gt;Model Training&lt;/a&gt;&lt;/span&gt;&lt;ul class=\"toc-item\"&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Train-SingleShotDetector-Model\" data-toc-modified-id=\"Train-SingleShotDetector-Model-3.1\"&gt;Train SingleShotDetector Model&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Find-the-good-learning-rate\" data-toc-modified-id=\"Find-the-good-learning-rate-3.2\"&gt;Find the good learning rate&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Train-the-model\" data-toc-modified-id=\"Train-the-model-3.3\"&gt;Train the model&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Unfreezing-the-backbone-and-fine-tuning\" data-toc-modified-id=\"Unfreezing-the-backbone-and-fine-tuning-3.4\"&gt;Unfreezing the backbone and fine-tuning&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Visualize-results\" data-toc-modified-id=\"Visualize-results-3.5\"&gt;Visualize results&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Save-and-load-trained-models\" data-toc-modified-id=\"Save-and-load-trained-models-3.6\"&gt;Save and load trained models&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;/ul&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Deploy-trained-model\" data-toc-modified-id=\"Deploy-trained-model-4\"&gt;Deploy trained model&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Model-management\" data-toc-modified-id=\"Model-management-5\"&gt;Model management&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Model-inference\" data-toc-modified-id=\"Model-inference-6\"&gt;Model inference&lt;/a&gt;&lt;/span&gt;&lt;ul class=\"toc-item\"&gt;&lt;li&gt;&lt;span&gt;&lt;a href=\"#Visualize-results\" data-toc-modified-id=\"Visualize-results-6.1\"&gt;Visualize results&lt;/a&gt;&lt;/span&gt;&lt;/li&gt;&lt;/ul&gt;&lt;/li&gt;&lt;/ul&gt;&lt;/div&gt;&lt;/del_n7bd&gt;</p>